channeldb/models/channel_edge_policy.go

```diff
@@ -198,3 +198,57 @@ func (c *ChannelEdgePolicy1) Sig() (input.Signature, error) {
 // A compile-time check to ensure that ChannelEdgePolicy1 implements the
 // ChannelEdgePolicy interface.
 var _ ChannelEdgePolicy = (*ChannelEdgePolicy1)(nil)
+
+type ChannelEdgePolicy2 struct {
+	lnwire.ChannelUpdate2
+
+	ToNode [33]byte
```

Comment:
Missing `godoc` comment. 

Also based on the comment above, I think we should also make these TLVs on disk in the db. Then we can just combine with the records above to serialize, maintaining extensibility for the future, and also re-using a ton of code to boot.